addons/zylann.hterrain/util/util.gd

```diff
@@ -57,6 +57,41 @@ static func create_wirecube_mesh(color = Color(1,1,1)) -> Mesh:
 	return mesh
 
 
+static func file_get_23(f: FileAccess) -> int:
+	var res : int
+	var a : int = f.get_8()
+	var b : int = f.get_8()
+	var c : int = f.get_8()
+
+	if f.big_endian:
+		var tmp : int = a
+		a = c
+		c = tmp
+
+	res = c
+	res <<= 8
+	res |= b
+	res <<= 8
+	res |= a
+
+	return res
+
+
+static func file_store_23(f: FileAccess, h: int) -> void:
```

Comment:
Should be `store_24`

Reply:
Thanks for catching that, that was just a brain derp